sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveCommandSuite.scala

```diff
@@ -290,6 +291,13 @@ class HiveCommandSuite extends QueryTest with SQLTestUtils with TestHiveSingleto
       checkAnswer(
         sql("SELECT employeeID, employeeName FROM part_table WHERE c = '2' AND d = '1'"),
         sql("SELECT * FROM non_part_table").collect())
+
+      // check for case insensitive of partition columns while querying.
+      withSQLConf(SQLConf.CASE_SENSITIVE.key -> "false") {
+        checkAnswer(
+          sql("SELECT employeeID, employeeName FROM part_table WHERE C = '2' AND D = '1'"),
+          sql("SELECT * FROM non_part_table").collect())
+      }
```

Comment:
This UT seems to succeed without your code. Please provide a valid UT to validate your contribution.

Reply:
sorry it was a mistake :)  i missed load data command where we shall specify the partition column name with different case compare to the actual provided while creation. thanks for pointing out.